packages/jellyfish-api-core/src/category/spv.ts

```diff
@@ -275,3 +328,28 @@ export interface CreateAnchorResult {
   /** decoded sendResult */
   sendMessage: string
 }
+
+export interface ListAnchorsResult {
+  /** BTC block height */
+  btcBlockHeight: number
+  /** BTC block hash */
+  btcBlockHash: string
+  /** BTC transaction hash */
+  btcTxHash: string
+  /** previous anchor */
+  previousAnchor: string
+  /** defi block height */
+  defiBlockHeight: number
+  /** defi block hash */
+  defiBlockHash: string
+  /** anchor reward address */
+  rewardAddress: string
+  /** BTC confirmations */
+  confirmations: number
+  /** number of signatures */
+  signatures: number
+  /** anchor status */
+  active?: boolean
```

Comment:
extra field? could not find this in c++ side

Reply:
ignore. https://github.com/DeFiCh/jellyfish/pull/618

Reply:
`spv_listanchorspending`, `spv_listanchorsunrewarded` and `spv_listanchors` rpcs are sharing same interface, only `spv_listanchors` contains `active` property